---
layout: page
title: SSX-MIER
header: Microbial Innovation and Ecosystems Research
permalink: /mier/
navigation_weight: 5
nav_name: MIER
---

## Overview
The Microbial Innovation and Ecosystems Research (MIER) group does stuff too.  

---
---

## Recent news  
#### The release of [GToTree: a user-friendly workflow for phylogenomics](https://github.com/AstrobioMike/GToTree/wiki)

<img src="/images/GToTree-Overview_slimmed.svg" style="float:right;width:450px;border:0px;box-shadow:none!important;margin:0px;padding:20px 0px 0px 10px">

> **Abstract**  
Genome-level evolutionary inference (i.e., phylogenomics) is becoming an increasingly essential step in many biologists' work - such as in the characterization of newly recovered genomes, or in leveraging available reference genomes to guide evolutionary questions. Accordingly, there are several tools available for the major steps in a phylogenomics workflow. But for the biologist whose main focus is not bioinformatics, much of the computational work required - such as accessing genomic data on large scales, integrating genomes from different file formats, performing required filtering, stitching different tools together, etc. - can be prohibitive. Here I introduce GToTree, a command-line tool that can take any combination of fasta files, GenBank files, and/or NCBI assembly accessions as input and outputs an alignment file, estimates of genome completeness and redundancy, and a phylogenomic tree based on the specified single-copy gene (SCG) set. While GToTree can work with any custom hidden Markov Models (HMMs), also included are 13 newly generated SCG-set HMMs for different lineages and levels of resolution, built based on searches of â‰ˆ12,000 bacterial and archaeal high-quality genomes. GToTree aims to give more researchers the capability to make phylogenomic trees.  



The program and documentation are available **[here](https://github.com/AstrobioMike/GToTree/wiki)**, and while the paper is in review, the preprint is available **[here](https://www.biorxiv.org/content/10.1101/512491v4)**.

---
---


## Available materials

[Publications]()  
[Presentations]()  

## People
